### DISCUSSION TITLE
Uploader button customization

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2683] [Q&A] [answered]
Any props to customize uploader component's button:
https://rsuitejs.com/components/uploader/

For example: i would like to change it's text to Change photo, and also give it orange color
Also is there any way how can i render uploading file?

**Comment by simonguo** [comment, marked as answer]
https://github.com/rsuite/rsuite/pull/2688 Waiting for the release of this feature.

**Reply by simonguo** [reply]
Supported in [v5.18.0](https://github.com/rsuite/rsuite/releases/tag/v5.18.0).
